keyboards/keychron/q5/ansi/keymaps/vial/Q5_US_ANSI_Knob_V1.4.json/Q5_US_ANSI_Knob_V1.4.json

```diff
@@ -0,0 +1,281 @@
+        "1,5",
+        "1,6",
+        "1,7",
+        "1,8",
+        "1,9",
+        "1,10",
+        "1,11",
+        "1,12",
+        {
+          "c": "#aaaaaa",
+          "w": 2
+        },
+        "1,13",
+        {
+          "x": 0.5
+        },
+        "1,15",
+        "1,16",
+        "1,17",
+        "3,14"
+      ],
+      [
+        {
+          "c": "#aaaaaa",
+          "w": 1.5
+        },
+        "2,0",
+        {
+          "c": "#cccccc"
+        },
+        "2,1",
+        "2,2",
+        "2,3",
+        "2,4",
+        "2,5",
+        "2,6",
+        "2,7",
+        "2,8",
+        "2,9",
+        "2,10",
+        "2,11",
+        "2,12",
+        {
+          "w": 1.5
+        },
+        "2,13",
+        {
+          "c": "#cccccc",
+          "x": 0.5
+        },
+        "2,15",
+        "2,16",
+        "2,17"
+      ],
+      [
+        {
+          "c": "#aaaaaa",
+          "w": 1.75
+        },
+        "3,0",
+        {
+          "c": "#cccccc"
+        },
+        "3,1",
+        "3,2",
+        "3,3",
+        "3,4",
+        "3,5",
+        "3,6",
+        "3,7",
+        "3,8",
+        "3,9",
+        "3,10",
+        "3,11",
+        {
+          "c": "#777777",
+          "w": 2.25
+        },
+        "3,13",
+        {
+          "c": "#cccccc",
+          "x": 0.5
+        },
+        "3,15",
+        "3,16",
+        "3,17",
+        {
+          "h": 2,
+          "y": -1
+        },
+        "2,14"
+      ],
+      [
+        {
+          "c": "#aaaaaa",
+          "y": 1,
+          "w": 2.25
+        },
+        "4,0",
+        {
+          "c": "#cccccc"
+        },
+        "4,2",
+        "4,3",
+        "4,4",
+        "4,5",
+        "4,6",
+        "4,7",
+        "4,8",
+        "4,9",
+        "4,10",
+        "4,11",
+        {
+          "c": "#aaaaaa",
+          "w": 1.75
+        },
+        "4,13",
+        {
+          "c": "#777777",
+          "x": 0.25,
+          "y": 0.25,
+          "w": 1
+        },
+        "4,14",
+        {
+          "c": "#cccccc",
+          "x": 0.25,
+          "y": -0.25
+        },
+        "4,15",
+        "4,16",
+        "4,17"
+      ],
+      [
+        {
+          "c": "#aaaaaa",
+          "w": 1.25
+        },
+        "5,0",
+        {
+          "w": 1.25
+        },
+        "5,1",
+        {
+          "w": 1.25
+        },
+        "5,2",
+        {
+          "c": "#cccccc",
+          "w": 6.25
+        },
+        "5,6",
+        {
+          "c": "#aaaaaa",
+          "w": 1
+        },
+        "5,10",
+        "5,11",
+        "5,12",
+        {
+          "c": "#777777",
+          "w": 1,
+          "x": 0.25,
+          "y": 0.25
+        },
+        "5,13",
+        "5,14",
+        "5,15",
+        {
+          "c": "#cccccc",
+          "x": 0.25,
+          "y": -0.25
+        },
+        "5,16",
+        "5,17",
+        {
+          "y": -1,
+          "h": 2
+        },
+        "5,5"
+      ]
+    ]
+  }
+}
```

Comment:
Considering this JSON is for the ANSI Knob version of the Q5, it wouldn't make sense to have this folder sit here since this PR mostly relates to the ANSI non-knob Q5.
```suggestion

```

Reply:
This file still needs to be deleted.